CONTRIBUTING.md

```diff
@@ -0,0 +1,619 @@
+
+programmer  
+: Do not use. Use *developer*.
+
+question marks ("?")  
+: Use questions sparingly. In general, user want us to give them answers. When a customer needs to make a decision, a question *may be* appropriate.
+
+*Bad example:*
+
+Why make TinyMCE mobile friendly?
+
+*Good examples:*
+
+- If you forgot your password, provide your secret answer.
+
+- More nature themes are available online.
+
+- Do you want to save your changes?
+
+quickstart, quick start  
+: Content to help developers get our products up and running quickly. Use *quickstart*.
+
+rich text editor, rich-text editor, RTE  
+: Most common way to refer to TinyMCE’s product category.
+
+Use rich text editor rather than rich-text editor (even if the later may be more technically correct). Do not use RTE.
+
+screen size, screensize  
+: Use *screen size*.
+
+SDK  
+: Do not use. Use *Self-hosted installation*.
+
+See or Refer to…  
+: Use *see* for internal links, use *visit* for external links.
+
+Such as: For information on creating links, see: &lt;link&gt;.
+
+self-hosted  
+: The preferred way to refer to an on-premises deployment of our
+technology.
+
+self-hosted premium plugins  
+: Preferred. Do not use *premium plugin SDKs*.
+
+should or must  
+: Avoid using *must* or *should* Only use where it is unavoidable.
+
+submenu  
+: Use *submenu* Do not use *sub-menu* or *sub menu*.
+
+Software as a Service (SaaS)  
+: Spell out on the first mention in a section for general audience. For a technical audience, it is always OK to use SaaS.
+
+Do not capitalize as SAAS.
+
+Do not hyphenate as a modifier, but include the abbreviated form in parentheses.
+
+spell checking, spell checker, spell check  
+: Do not use the compound words *spellcheck* or *spellchecker*
+
+Do not hyphenate.
+
+textarea, text box, textbox  
+: Use *textarea* to refer to a multi-line text box in a form.
+
+Tiny  
+: Use to refer to the company or our entire solution/platform.
+
+Do not use *Tiny* to refer to the core editor, which is *TinyMCE*.
+
+Tiny 4.x, Tiny 5.0  
+: *Tiny* is not a product. Do not give it a version. Use the product name, such as TinyMCE 5.0 or TinyMCE 4.9.
+
+Tiny Cloud  
+: Correct way to refer to the Tiny Cloud service. Spell it out on all occasions.
+
+For example: *Tiny Cloud* includes a fast content delivery network.
+
+Tiny Technologies Inc.  
+: *Tiny Technologies Inc.* is full and correct business name for *Tiny*
+
+Do not use *Ephox Corporation*.
+
+TinyMCE  
+: This is the JavaScript library for rich text editing.
+
+Capitalize the *MCE* Do not use *Tinymce* *tinymce* is allowed in code
+and codeblocks where required.
+
+TinyMCE 4  
+: Correct way to refer to the 4.x releases of TinyMCE unless a specific version number (major.minor.patch) is *required*.
+
+TinyMCE 4.x  
+: Do not use. Use *TinyMCE 4* or *version 4* If necessary to specify the availability of a new feature or change in behavior, specify the minor version number such as TinyMCE 4.7.
```

Comment:
```suggestion
: Do not use. Use *TinyMCE 5* or *version 5* If necessary to specify the availability of a new feature or change in behavior, specify the minor version number such as TinyMCE 5.7.
```

Update the glossary’s version number examples to reflect the release of *TinyMCE 6*.